catalogd/internal/storage/localdir_test.go

```diff
@@ -322,6 +365,12 @@ func TestQueryEndpoint(t *testing.T) {
 			expectedStatusCode: http.StatusOK,
 			expectedContent:    `{"image":"quaydock.io/namespace/bundle:0.0.3","name":"bundle.v0.0.1","package":"webhook_operator_test","properties":[{"type":"olm.bundle.object","value":{"data":"dW5pbXBvcnRhbnQK"}},{"type":"some.other","value":{"data":"arbitrary-info"}}],"relatedImages":[{"image":"testimage:latest","name":"test"}],"schema":"olm.bundle"}`,
 		},
+		{
+			name:               "valid query with multi-json, json-lines formatted response",
+			queryParams:        "?schema=olm.channel",
+			expectedStatusCode: http.StatusOK,
+			expectedContent:    `{"entries":[{"name":"bundle.v0.0.1"}],"name":"preview_test","package":"webhook_operator_test","schema":"olm.channel"}` + "\n" + `{"entries":[{"name":"multi-bundle.v0.0.1"}],"name":"review_test","package":"multi_operator_test","schema":"olm.channel"}`,
+		},
```

Comment:
Fyi I'm adding a similar test with [this PR](https://github.com/operator-framework/operator-controller/pull/1713/files#diff-e9e3717a08766199e1050c87e767901da6c86b5e90be548167585f253315869cR332-R337)

Reply:
So this can be removed.